lib/cpp/src/thrift/server/TConnectedClient.cpp

```diff
@@ -86,7 +86,8 @@ void TConnectedClient::run() {
     } catch (const TException& tex) {
```

Comment:
Just wondering, is catching TException enough here or should we catch std::exception here instead?